Makefile

```
@@ -1,10 +1,25 @@
-.PHONY: install test clean
+.PHONY: fmt lint test generate-lockfile clean
```

```suggestion
.PHONY: fmt lint test generate-lockfiles clean
```

Minor: Think this is a typo/copypaste issue